app/views/core_induction_programmes/mentor_materials/index.html.erb

```diff
@@ -11,10 +11,7 @@
       <% if cip.mentor_materials.any? %>
       <% cip.mentor_materials.each do |mentor_material| %>
         <p><%= govuk_link_to mentor_material.title, mentor_material_path(mentor_material) %></p>
-        <% if mentor_material.course_lesson %>
-          <p>Corresponding lesson: <%= govuk_link_to mentor_material.course_lesson.title, lesson_path(mentor_material.course_lesson) %></p>
-          <% else %> <p>No corresponding lesson found</p>
-       <%end %>
+        <p>Corresponding lesson: <%= govuk_link_to mentor_material.course_lesson.title, lesson_path(mentor_material.course_lesson) %></p>
       <% end %>
       <% else %>
       <p>No mentor materials were found!</p>
```

Comment:
Would it be worth just removing index completely? I know i've not touched it since we added links elsewhere

Reply:
I touched it quite a few times - whenever there is a ticket saying 'do something for each mentor material for a particular CIP'.